src/lit-element.ts

```diff
@@ -70,26 +87,26 @@ export function styleString(styleInfo: any) {
   return o.join('; ');
 }
 
-export class LitElement extends PropertiesMixin(HTMLElement) {
+export class LitElement extends PropertiesMixin
+(HTMLElement) {
 
   private __renderComplete: Promise<any>|null = null;
   private __resolveRenderComplete: Function|null = null;
   private __isInvalid: Boolean = false;
   private __isChanging: Boolean = false;
+  private _root?: Element|DocumentFragment;
 
-  protected ready() {
+  ready() {
     this._root = this._createRoot();
     super.ready();
   }
 
-
-
   /**
    * Returns an
-   * @returns {Node|ShadowRoot} Returns a node into which to render.
-  */
-  protected _createRoot() {
-    return this.attachShadow({mode: 'open'});
+   * @returns {Element|DocumentFragment} Returns a node into which to render.
```

Comment:
Types can (and IMO should) be left out of jsdoc in ts files

Reply:
done